### PR TITLE
Added NuGet-section to ReadMe

### PR DESCRIPTION
**Pull request by gfoidl**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -29,3 +29,25 @@ namespace Sample
 
 - Generates UTF-8 binary data from string literals (UTF-16).
 - The UTF-8 data is optimized to avoid allocation. see: [C# ReadOnlySpan and static data](https://vcsjones.dev/2019/02/01/csharp-readonly-span-bytes-static/)
+
+## NuGet
+
+[![NuGet](https://img.shields.io/nuget/v/StringLiteralGenerator?style=flat-square)](https://www.nuget.org/packages/StringLiteralGenerator)
+
+You may also need to add a reference to `Microsoft.Net.Compilers.Toolset`.
```
Comment (ufcpp) [inline readme.md:37]: Is `Microsoft.Net.Compilers.Toolset` needed even when using .NET 5 SDK RC1 or later?

Comment (gfoidl) [reply readme.md:37]: I tried these:

| SDK | needed |
| --- | --- |
| RC 2 | no |
| RC 1 | didn't test |
| previews | yes |

That's why I've written `may`. 
Is it better to explicitely call this out, like
```suggestion
For versions earlier than .NET 5 SDK RC2 you may also need to add a reference to `Microsoft.Net.Compilers.Toolset`.
```
?

**Comment by ufcpp** [comment]
Thanks a lot.

**Comment by gfoidl** [comment]
I have to thank you for the very useful source generator 👍

**Pull request merged**